### PR TITLE
Update oasisengine.json

### PR DESCRIPTION
**Pull request by eyworldwide**
- Add a new `0.5` to  version variable;
- Remove `sitemap_urls`  because the `sitemap.xml` only contains the newest version urls and I can't search the documentation according to the version facet. I wonder if it's a good idea. I expect the crawler will crawl all the urls in the site.

### CODE REVIEW

File: configs/oasisengine.json [hunk 1]
```diff
@@ -16,15 +17,13 @@
...
+          "0.4",
+          "0.5"
         ]
       },
       "page_rank": 1
     }
   ],
-  "sitemap_urls": [
-    "https://oasisengine.cn/sitemap.xml"
-  ],
```
Comment (shortcuts) [inline configs/oasisengine.json:27]: Having sitemaps is always better, even if they don't contain all the URLs it's still a good entry point for the crawler

Comment (eyworldwide) [reply configs/oasisengine.json]: Thank you for your advice. I've recovered the code.

Comment (shortcuts) [reply configs/oasisengine.json]: Feel free to merge the master branch so I can merge

Comment (eyworldwide) [reply configs/oasisengine.json]: @shortcuts Pls merge to master branch🥳

Comment (shortcuts) [reply configs/oasisengine.json]: @eyworldwide it seems I'm not able to merge master in your PR due to some collaborating settings on your GH account

Comment (eyworldwide) [reply configs/oasisengine.json]: @shortcuts I'll check the problem first. If it not works, I will try to pull another request through my personal account.

Comment (eyworldwide) [reply configs/oasisengine.json]: I've pulled another PR from a new branch: https://github.com/algolia/docsearch-configs/pull/4556

**Comment by eyworldwide** [comment]
Thank you! @shortcuts

**Pull request closed without merging**